definitions/daily-standup/README.md

```diff
@@ -0,0 +1,35 @@
+---
+permalink: /definitions/daily-standup/
+---
+
+# Definition of the Daily Standup
+
+> _“Dreams are, by definition, cursed with short life spans.”_
+> ~ Candice Bergen
+
+## Introduction
+
+Standup is a thing we use to inform each other on blockers or impediments. 
+We use this to share information and quickly interfere if someone needs / might 
+need help. We excluded the Product Owners from the stand-up (for now). 
+
+The product Owners can "stand (up)" with us but are not allowed to ask for updates. 
```

Comment:
`us` should be `the development team` (as "we" earlier refers to Dealerdirect).

Also, either "product" should be capitalized or "Owners" should not.